README.md

```diff
@@ -10,10 +10,21 @@ Clone this repository and install the dependencies.
 
 ## Testing
 
-Runs integration tests using arlocal. Note - these test write temporary files to your local disc.
+Unit tests and integration tests are located in the [tests] directory. Both unit and integration tests are written using [Jest].
 
-- `yarn test` - runs full test suite
-- `yarn test -t 'auctions'` - runs tests that match this spec name (e.g. `auctions.test.ts`)
+### Integration Tests
+
+Runs integration tests against deployed Smartweave contract using [Warp SDK].
+
+- `yarn test:integration` - runs full test suite
+- `yarn test:integration -t 'auctions'` - runs tests that match this spec name (e.g. `auctions.test.ts`)
+
+### Unit Tests
+
+Runs unit tests against contract source code located in [src] directory.
+
+- `yarn test:unit` - runs full unit test suite located in [tests/unit]
```

Comment:
I think this pathing pattern may break convention with a number of our other repos where unit test files are colocated with source files (e.g. myfile.ts and myfile.test.ts in the same folder). Makes for easier project navigation. Integration tests do tend to live in another folder though.

Reply:
our setup for jest tests live in `tests/` directory so i kept them colacted to the jest setup, but can move to `src` directory

Reply:
moved to follow existing patterns in this commit 

[d067fd5](https://github.com/ar-io/testnet-contract/pull/113/commits/d067fd519f5d442754a6eb27d7981d5f5675761c)